src/hotspot/share/runtime/sharedRuntime.cpp

```diff
@@ -1881,6 +1880,12 @@ address SharedRuntime::handle_unsafe_access(JavaThread* thread, address next_pc)
   return next_pc;
 }
 
+#ifdef JFR_HAVE_INTRINSICS
+JRT_LEAF(void, SharedRuntime::trace_id_load_barrier(Klass * klass))
```

Comment:
This moves to "jfr/jfr.cpp" instead, thanks.

Reply:
fixed.